Validation/HcalDigis/interface/HcalDigisClient.h

```diff
@@ -78,6 +78,21 @@ class HcalDigisClient : public DQMEDHarvester {
         msm_->find(name)->second->Fill(X, weight);
     }
 
+    //TP Code
+    void projectY(std::string name_to, TH2F* TH2F_from){
```

Comment:
moreover if you are interested in mean and its error you should probably use a TProfile directly instead of a TH2F